Formula/threesdk.rb

```diff
@@ -0,0 +1,140 @@
+class Threesdk < Formula
+    desc "Software development kit for JumpscaleX"
+    homepage "https://sdk.threefold.io/"
+    url "https://github.com/threefoldtech/jumpscaleX_core/archive/v10.5.2.tar.gz"
+    sha256 "d2ce77a61a198b2cd53f4acce5fd7e5c36c369fc83a05b9f74314530b4f4a175"
+
+    depends_on "python@3.8" => :build
```

Comment:
Is this really only needed to build the software?

Reply:
yes using pyinstaller to package the binary

Reply:
in that case it can just use 
```suggestion
    depends_on "pyinstaller" => :build
```

Reply:
Then no need for it at all pyinstaller is installed from the resources, it's a pip

Reply:
If something is installable through homebrew you must use that version, rather than resourcing your own.

Reply:
yes add it with the hash and md5